client/src/components/lobby/Options.vue

```diff
@@ -121,20 +121,11 @@ import { useGameStore } from '@/stores/game.js';
 import { mapState } from 'pinia';
 import { defineComponent } from 'vue';
 
-type Options = {
-  promptTimer: number;
-  numRounds: number;
-  autoNumRounds: boolean;
-  promptSkipping: boolean;
-  sikeDispute: boolean;
-  sikeRetries: number;
-  packs: Record<string, boolean>;
-  customPrompts: string;
-};
+import { SettableOptions } from ':common/options';
```

Comment:
For this and all imports from :common, do you need to declare type? i.e `import { type SettableOptions }` ? Is there any reason why this has to be different from how you are importing `:common/player` or `:common/match`?

Reply:
On the client side I had a linter setting or typescript setting that expected explicit type imports. Not sure how it slipped through, maybe that setting has been turned off.